docs/running-on-mesos.md

```diff
@@ -479,6 +479,35 @@ See the [configuration page](configuration.html) for information on Spark config
 </tr>
 
 <tr>
+  <td><code>spark.mesos.driver.secret.envkey</code></td>
+  <td><code>(none)</code></td>
+  <td>
+    If set, the contents of the secret referenced by
+    spark.mesos.driver.secret.name will be written to the provided
+    environment variable in the driver's process.
+  </td>
+  </tr>
+  <tr>
+<td><code>spark.mesos.driver.secret.filename</code></td>
+  <td><code>(none)</code></td>
+  <td>
+    If set, the contents of the secret referenced by
+    spark.mesos.driver.secret.name will be written to the provided
+    file.  Relative paths are relative to the container's work
+    directory.  Absolute paths must already exist.  Consult the Mesos Secret
+    protobuf for more information.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.mesos.driver.secret.name</code></td>
```

Comment:
Ah you're right, we'll need a separate system for value secrets. I would imagine that `VALUE` secrets are mainly used as environment-based secrets? Let me re-do this.